### DISCUSSION TITLE
Chain relaxation and short MD

### DISCUSSION THREAD

**Discussion by joaomcteixeira** [THGLab/MCSCE#7] [Implementations]
At which steps does the software require chain minimizations and relaxations performed by external packages? Which packages do the original software use for that and which possible replacements are, if so, desirable? My knowledge of MD packages is very reduced, but we may consider using OpenMM because it has a well-developed Python interface. But, likely, there are others that also have.

**Comment by JerryJohnsonLee** [comment]
I don't remember there is a chain minimization / MD step in the original implementation. I believe it is just growing sidechains from the backbone and that's it. We can also add this relaxation step in the reimplementation if we find that is necessary/helpful, and openMM sounds good to me.

**Reply by joaomcteixeira** [reply]
Okay. So, we consider this settled until we find the need to incorporate minimizations. If that time comes, we will reopen this discussion line.
